Miniforge3/construct.yaml

```diff
@@ -1,16 +1,16 @@
 name: Miniforge3
+version: 4.7.11-0
```

Comment:
You didn't like the tag idea?

Reply:
After discussing with @isuruf, I found it more natural to hard-code the version in the repo instead of in a git tag. It's more visible but most importantly it allows easy bump of version in a PR without having to create a git tag.

Reply:
You will have to create a tag to create a release on Travis and GitHub. Either way works,we just have to remember to make them match

Reply:
Sure creating a tag that match the yaml version is fine.

Reply:
I will add some instructions about releasing.